### DISCUSSION TITLE
nextcloud image should get optimized

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2404] [Ideas] [answered]
Currently, the unpacked Nextcloud image is around 400 MB big
![grafik](https://user-images.githubusercontent.com/75573284/233684801-9ede5570-6a97-499d-af7a-61df01646828.png)

After running this Dockerfile (so copying the plain data)
```Dockerfile
FROM nextcloud/aio-nextcloud:latest as nc
FROM scratch
COPY --from=nc / /
```
it looks like this:
![grafik](https://user-images.githubusercontent.com/75573284/233684970-9999e1f7-3ab3-4d3c-9d0a-175500755585.png)

Maybe the Dockerfile could be improved by doing anything inside one RUN step?

**Comment by Zoey2936** [comment]
Size with one RUN step:
![grafik](https://user-images.githubusercontent.com/75573284/233687046-b51e102b-c092-4630-a292-c2f96865e957.png)

**Comment by Zoey2936** [comment]
@szaimen what's your opinion, should I create a PR?

**Reply by szaimen** [reply, marked as answer]
Yes, makes probably sense to look into improving that. If possible retaining the currently structure but merging some steps would be welcome. A PR would be great! :)
